packages/cli/src/Queue.ts

```diff
@@ -62,24 +62,20 @@ export class Queue {
 		this.jobQueue.on('global:progress', (jobId, progress: WebhookResponse) => {
 			this.activeExecutions.resolveResponsePromise(
 				progress.executionId,
-				WebhookHelpers.decodeWebhookResponse(progress.response),
+				decodeWebhookResponse(progress.response),
 			);
 		});
 	}
 
-	async add(jobData: JobData, jobOptions: object): Promise<Job> {
-		return this.jobQueue.add(jobData, jobOptions);
-	}
-
-	async getJob(jobId: JobId): Promise<Job | null> {
-		return this.jobQueue.getJob(jobId);
-	}
```

Comment:
instead of removing these, can we update the code elsewhere to use these methods instead of using the `jobQueue` directly?

Reply:
do these methods offer any value? they don't do any additional checks or operations but just tunnel the method call through the service

Reply:
I'm working on adding [metrics for queue](https://github.com/n8n-io/n8n/pull/7263) for auto-scaling, and in that branch, it was helpful to have all bull specific code in a single class, instead of being spread across multiple files.